samples/scheduling/date-range-picker/custom-ranges/src/index.ts

```diff
@@ -0,0 +1,47 @@
+import { defineComponents, IgcDateRangePickerComponent } from 'igniteui-webcomponents';
+import { CustomDateRange } from 'igniteui-webcomponents/components/date-range-picker/date-range-picker.js';
+import 'igniteui-webcomponents/themes/light/bootstrap.css';
+
+defineComponents(IgcDateRangePickerComponent);
+
+export class DateRangePickerCustomRanges {
+  constructor() {
+
+    const dateRange = document.querySelector('igc-date-range-picker') as IgcDateRangePickerComponent;
+    const today: Date = new Date();
+
```

Comment:
```suggestion
    const today = new Date();

```
minor, typescript can infer the type with this assignment